msteams-platform/apps-in-teams-meetings/build-apps-for-anonymous-user.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Build apps for anonymous user
+author: v-sdhakshina
+description: In this article, learn how to test the experience delivered to the anonymous users in meeting apps with all admin settings.
+ms.topic: conceptual
+ms.author: v-sdhakshina
+ms.localizationpriority: medium
+---
+
+# Build apps for anonymous users
+
+You can test the experience delivered to anonymous users by selecting the meeting join URL present in the meeting invite and joining the meeting via private window.
```

Comment:
Do we show the actual bot icon for anonymous users, or a placeholder? There's a bug filed reporting this (generic "B" icon in hexagon). If it's by design let's mention it here with a screenshot, so that our support teams have a page to reference when partners report this.

Reply:
@v-sdhakshina add para 
Message sent with Bot app does not display app icon for anonymous user, instead a generic icon is displayed, see below:
![image](https://user-images.githubusercontent.com/92553522/194410560-edac361e-9f67-46cc-924f-a064277a9603.png)